Gemfile

```diff
@@ -41,9 +41,15 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rubocop', require: false
 end
 
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 gem "active_model_serializers", "~> 0.10.2"
+
+# For integration with github
+gem "github_api"
+# For Fake data
```

Comment:
Can you explained why you used an unreleased version of Faker?

Reply:
With **faker** this uses the the `1.9.3` version so i added there
Now change with `gem 'faker', '~> 1.6', '>= 1.6.6'`